CHANGELOG.md

```diff
@@ -4,6 +4,28 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v11.0.0](https://github.com/voxpupuli/puppet-redis/tree/v11.0.0) (2024-05-07)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-redis/compare/v10.0.0...v11.0.0)
+
+**Breaking changes:**
+
+- Drop EL7 compatibility code [\#509](https://github.com/voxpupuli/puppet-redis/pull/509) ([ekohl](https://github.com/ekohl))
```

Comment:
now should we drop CentOS 8 as well?

Reply:
I guess we could?

Reply:
https://github.com/voxpupuli/puppet-redis/pull/525